static/panes/cfg-view.ts

```diff
@@ -169,6 +157,21 @@ export class Cfg extends Pane<CfgState> {
         return 'CFG';
     }
 
+    override registerButtons() {
```

Comment:
Moving the selector setup to `registerButtons` to fix a race condition where `onCompileResult` can fire before the cfg view constructor sets `this.functionSelector`